packages/web/src/scss/components/UNSTABLE_Avatar/README.md

```diff
@@ -0,0 +1,101 @@
+# UNSTABLE Avatar
+
+⚠️ This component is UNSTABLE. It may significantly change at any point in the future.
+Please use it with caution.
+
+The `UNSTABLE_Avatar` component is used to represent a user or entity.
+
+It can be a circle or a square (with rounded corners, using `radius-100` token) and
+can have different sizes.
+
+## Example
+
+```html
+<div class="UNSTABLE_Avatar UNSTABLE_Avatar--medium" aria-label="Profile of Jiří Bárta">
+  <span aria-hidden="true">JB</span>
+</div>
+
+<a href="#" class="UNSTABLE_Avatar UNSTABLE_Avatar--xsmall" aria-label="Profile of Jiří Bárta">
+  <img src="https://picsum.photos/id/823/162/162" alt="Jiří Bárta" aria-hidden="true" />
+</a>
+```
+
+## Square
+
+Add `UNSTABLE_Avatar--square` modifier to make the avatar a square.
```

Comment:
I'd rather see this as two modifiers: `--square` and `--circle`. Then we could set it through a prop like this:

```jsx
<Avatar shape="square" />
```

`circle` would be considered default, so you would have to add the `--circle` modifier in the `web` package for default usage, just like you set the size.

Reply:
Whether we end up with two modifiers or not, we should keep the same approach in React too.

https://github.com/lmc-eu/spirit-design-system/pull/1477#discussion_r1647671095